src/components/RecentlyOpened.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="recently-opened">
+    <span>Recently Opened:</span>
+    <div class="recent-items">
+      <a v-for="path in paths" v-bind:title="path.path" :key="path.path" @click="$emit('selected', path.path)">
+        {{ path.display }}
+      </a>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapActions } from 'vuex';
+
+export default {
+  data() {
+    return {
+      paths: []
+    };
+  },
+  created() {
+    this.getRecentlyOpened().then(paths => {
+      paths = paths.splice(0, 2); // show only 2 paths
+      this.paths = paths.map(path => {
+        const parts = path.split('/');
+        return {
+          path,
+          display: parts[parts.length - 1]
+        };
+      });
+    });
+  },
+
+  methods: {
+    ...mapActions(['getRecentlyOpened'])
+  }
```

Comment:
@emanlodovice Can we make recently opened array included in the store's state? Then the `created` method can then become a computed property based on the state.

Reply:
thanks for the review. updated @arnellebalane . please check.